srcs/client/src/Components/Tools/NavBar/NavBar.tsx

```diff
@@ -100,57 +100,150 @@ export default function NavBar ()
+                            </div>
+                        </ul>
+                
+            </div>
+            </div>
+        </nav> */}
+
+
+
+
+    //     <Navbar className="navbar" bg="transparent" variant="transparent"sticky="top" expand="md" collapseOnSelect>
+    //     <Navbar.Brand>
+    //         <Link to="/"><h2 className="blueText" data-testid="HomeLink"> PONG</h2></Link>
+    //     </Navbar.Brand>
+    //     <Navbar.Toggle className="coloring" />
+    //         <Navbar.Collapse>
+    //         <Nav>
+    //             <Nav.Link >
+    //                 <Link to="/" className="">   <h2 className="yellowText" > Home </h2></Link>
+    //             </Nav.Link>
+    //             <Nav.Link >
+    //                 <Link to="/leaderboard"> <h2 className="yellowText" data-testid="LeaderboardLink">Leaderboard</h2> </Link>
+    //             </Nav.Link>
+    //             <Nav.Link >
+    //                 <Link to="/login">  <h2 className="yellowText" > Login </h2> </Link>
+    //             </Nav.Link>
+    //             <NavDropdown title="LOGIN">
+    //                 <NavDropdown.Item href="#products/profile">Profil</NavDropdown.Item>
+    //                 <NavDropdown.Item href="#products/chocolate">Edit Profil</NavDropdown.Item>
+    //                 <NavDropdown.Divider />
+    //                 <NavDropdown.Item href="#products/promo">Disconnect</NavDropdown.Item>
+    //             </NavDropdown>
+    //         </Nav>
+    //         </Navbar.Collapse>
+    // </Navbar>
+
+
+    // FORMER CO
+    // <nav className="navbar navbar-expand-md bg-dark">
+    //             <div className="">
+    //                 <div className="buttonInNavBar">
+    //                     <Link  to="/"> <h2 className="blueText">PONG</h2> </Link>
+    //                 </div>
+    //                 <div className="buttonInNavBar">
+    //                     <Link  to="/home"> <h2 className="yellowText"  >Home</h2> </Link>
+    //                 </div>
+    //                 <div className="buttonInNavBar">
+    //                     <Link  to="/about"><h2 className="yellowText" >About</h2> </Link>
+    //                 </div>
+    //                 <div className="buttonInNavBar">
+    //                     <Link  to="/chat">  <h2 className="yellowText" >Chat</h2> </Link>
+    //                 </div>
+    //                 <div className="buttonInNavBar">
+    //                     <Link  to="/leaderboard">  <h2 className="yellowText" >Leaderboard</h2> </Link>
+    //                 </div>
+    //                 <div className="buttonInNavBar">
+    //                     <button onClick={deco} className="playFlickering">Logout</button>
+    //                 </div>
+    //                 <div className="buttonInNavBar">
+    //                     <Link  to="/profile" state={{userID}}> <ProfilNavBar /> </Link>
+    //                 </div>
+    //             </div>
+    //       </nav>
```

Comment:
I think it is better to clean the 'old code' in files when pushing to master.
If you want to keep a copy of an old code just in case, you can create a branch before applying new modifications so the original branch will save a copy of the old code.